dockers/docker-snmp/snmp_yml_to_configdb.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/lib/env python3
```

Comment:
The shebang should read `#!/usr/bin/env python3`

Reply:
fixed.